src/Time/Time.php

```diff
@@ -16,118 +16,111 @@
 
 namespace Zest\Time;
 
+use Zest\Common\TimeZone;
 use Zest\Contracts\Time\Time as TimeContract;
 
 class Time implements TimeContract
 {
     /**
-     * Get "now" time
+     * Get "now" time.
      *
      * @param string $timezone Valid php supported timezone.
      *
      * @since 3.0.0
      *
-     * @return  int
+     * @return int
      */
-    public static function now($timezone = NULL)
+    public static function now($timezone = '')
```

Comment:
The `$timezone` will be the `string` type, not `null`.

And it makes sense that let the `$timezone` be empty string by default.